examples/daal/cpp_sycl/source/k_nearest_neighbors/bf_knn_dense_regression_batch.cpp

```diff
@@ -0,0 +1,218 @@
+/* file: bf_knn_dense_regression_batch.cpp */
+/*******************************************************************************
+ * Copyright 2014-2020 Intel Corporation
```

Comment:
```suggestion
 * Copyright 2020 Intel Corporation
```

Reply:
This header was just copied from another example.